addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -69,3 +75,84 @@ test("editing livechat note is synced between tabs", async () => {
     await document.querySelector(".o-livechat-ChannelInfoList textarea").blur(); // Trigger the blur event to save the note
     await waitFor(".o-livechat-ChannelInfoList textarea:value('Updated note')", { target: tab1 }); // Note should be synced with bus
 });
+
+test("livechat status is displayed in discuss sidebar", async () => {
```

Comment:
Would rename _"shows live chat status in discuss sidebar"_.

This is the same, but this wording is clearer on intention and scope of test.